### DISCUSSION TITLE
Define component level tokens

### DISCUSSION THREAD

**Discussion by peterbartels** [chakra-ui/panda#2220] [Q&A]
How can I define and use component level (semantic) tokens in Panda CSS?

For example `divider-thickness-small` or `card-background-primary`?

**Comment by kiruushaaa** [comment]
Did you mean that?
[playground](https://play.panda-css.com/BL5L9uClNQ)
[panda docs](https://panda-css.com/docs/theming/tokens#token-nesting)
<img width="640" alt="image" src="https://github.com/chakra-ui/panda/assets/18754878/a4869c01-038f-4681-916a-53ba12aa091e">

**Reply by peterbartels** [reply]
I want to use a custom structure as well so not put the `card` in the `color` but rather make `card` the root of a token structure, and then use: `{card.background.primary}`

**Reply by kiruushaaa** [reply]
As far as i remember you can use `bg: 'card.background.primary'` without functions but in a case a property isn't complex. If it not the key I assume we should wait for help from core guys

ps. Recipes _(maybe slot ones?)_ seem to be more convenient 🤔

**Reply by peterbartels** [reply]
Yes @kiruushaaa  ` card.background.primary` works because it's reading it from colors. 

To put it differently I would be nice to define a custom schema. For example Spectrum has a [component based token schema ](https://github.com/adobe/spectrum-tokens/blob/main/packages/tokens/src/layout-component.json) where tokens like this structure are defined: `checkbox-control-size-large` . 

However as I can see it's not possible to add them to the current token preset schema of panda.
